nf_core/lint/actions_schema_validation.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import logging
+import yaml
+import json
+import jsonschema
+import os
+import requests
+
+
+def actions_schema_validation(self):
+    """
+    Validate workflows against a schema
+
+    Uses the JSON Schema from
+    https://json.schemastore.org/github-workflow
+    to verify that all workflow yml files adhere to the correct Schema
+    """
+    passed = []
+    warned = []
+    failed = []
+
+    # Only show error messages from schema
+    logging.getLogger("nf_core.schema").setLevel(logging.ERROR)
+
+    # Get all workflow files
+    action_workflows = os.listdir(os.path.join(self.wf_path, ".github/workflows"))
```

Comment:
Maybe worth only getting `.yml` / `.yaml` files? In case any other auxiliary stuff is dumped there..

Reply:
Yeah thought about this for a minute but then figured there should only be workflow files in there ... but you're right, better safe than sorry :)